Scroller/EditableWrapper.js

```diff
@@ -165,8 +168,15 @@ const EditableWrapper = (props) => {
 
 				// Set the selected item's offset to css variable
 				const offset = (toIndex - fromIndex) * itemWidth;
-				wrapperRef.current.style.setProperty('--selected-item-offset', offset + 'px');
-
+				if (trigger === 'scroll') {
+					if (toIndex === dataSize - 1) scrolloffset = offset;
+					if (toIndex === 0) scrolloffset = (-1) * itemWidth * rearrangedItems.length;
```

Comment:
We don't use one line if statement without brace.
```suggestion
					if (toIndex === dataSize - 1) {
						scrolloffset = offset;
					}
					if (toIndex === 0) {
						scrolloffset = (-1) * itemWidth * rearrangedItems.length;
					}
```

Reply:
As I changed to logic, it's removed.